src/main.rs

```diff
@@ -259,7 +259,6 @@ pub(crate) async fn run_encrypted_payload(
     )?;
 
     optional_unzip_payload(&unzipped, config)?;
-
```

Comment:
This empty space removal is likely unrelated but I don't see an issue with it :smile:

Reply:
I think that this comes from "eglot-format", and call "cargo fmt".  Do you want me to remove it?

Reply:
No, I think it's fine :+1: